packages/vm/tests/BlockchainTestsRunner.ts

```diff
@@ -119,14 +119,18 @@ export default async function runBlockchainTest(options: any, testData: any, t:
       common.setHardforkByBlockNumber(currentBlock.toNumber())
 
       if (raw.transactionSequence) {
+        const parentBlock = await vm.blockchain.getIteratorHead()
+        const blockBuilder = await vm.buildBlock({
+          parentBlock,
+          headerData: { coinbase: '0x96dc73c8b5969608c77375f085949744b5177660' },
```

Comment:
i don't think we need `{ headerData: coinbase }` or `{ blockOpts: freeze }` so we can simplify by removing those opts

Reply:
Removed it, I indeed copied it from a buildBlock test! :smile:

Reply:
hm i'm still seeing it

Reply:
i'll remove it